griptape/structures/structure.py

```diff
@@ -20,7 +21,7 @@
 
 
 @define
-class Structure(ABC, RuleMixin):
+class Structure(RuleMixin, RunnableMixin["Structure"], ABC):
```

Comment:
Dang, didn't realize you need to explicitly specify that generic argument. Still nice reuse though.

Reply:
@dylanholmes was able to get rid of this